crates/ironrdp-server/src/server.rs

```diff
@@ -180,6 +190,25 @@ impl RdpServer {
         Ok(())
     }
 
+    async fn handle_input_backlog(&mut self, frames: Vec<Vec<u8>>) -> Result<()> {
+        for frame in frames {
+            match Action::from_fp_output_header(frame[0]) {
+                Ok(Action::FastPath) => {
+                    let input = FastPathInput::from_buffer(Cursor::new(&frame))?;
+                    self.handle_fastpath(input).await;
+                }
+
+                Ok(Action::X224) => {
+                    let _ = self.handle_x224(&frame).await;
+                }
+
+                _ => unreachable!(),
```

Comment:
Are we sure this is really unreachable? My understanding is that we are accumulating the additional frames as-is in the connector, did I miss something?

Reply:
If `Action::from_fp_output_header` could return `Err`, that would mean the frame isn't valid, but then it would have already failed in the acceptor loop, because `find_size` would have failed.

Reply:
You’re right! Do you think you could copy put this comment as an inline code comment above the `unreachable!` statement, for future reference?